lib/tori/backend/s3.rb

```diff
@@ -91,6 +91,17 @@ def read(filename)
         body(filename).read
       end
 
+      def public_url(filename)
+        "#{client.config.endpoint}/#{@bucket}/#{filename}"
+      end
+
+      def url_for(filename, method)
+        signer = Aws::S3::Presigner.new(client: client)
+        signer.presigned_url(method, bucket: @bucket, key: filename)
+      end
+
+      private
+
```

Comment:
Ah...I know that special methods are needed. However, I don't understand the reason that you use special methods. I think it's possible by common methods.  Do you have any reason?